tests/generate_qflags_tests.py

```diff
@@ -0,0 +1,69 @@
+from typing import Tuple, List
+
+SOURCE = 'windowFlags.py'
+MARKER_SPECIFIC_START = '### Specific part'
+MARKER_SPECIFIC_END = '### End of specific part'
+
+# Content is:
+# - name of the file to generate
+# - name of the QFlags<> class
+# - name of the individual flag class (inheriting from int)
+# - one individual flag value
+# - a second individual flag value
+FLAGS_FILES = [
+    ('qflags_alignmentflags.py', 'QtCore.Qt.Alignment', 'QtCore.Qt.AlignmentFlag', 'QtCore.Qt.AlignLeft', 'QtCore.Qt.AlignRight'),
+
+]
```

Comment:
There would be other changes to go with this.  I prefer to use code to document things over comments when possible.

```suggestion
@dataclasses.dataclass
class FlagsFileNames:
    generated_file: str
    flags_class: str
    flag_class: str
    first_flag_value: str
    second_flag_value: str

FLAGS_FILES = [
    FlagsFileNames(
        generated_file='qflags_alignmentflags.py',
        flags_class='QtCore.Qt.Alignment',
        flag_class='QtCore.Qt.AlignmentFlag',
        first_flag_value='QtCore.Qt.AlignLeft',
        second_flag_value='QtCore.Qt.AlignRight',
    ),
]
```

Reply:
This code is gone now, the super scripts replaces it.